### ISSUE TITLE
Retrospective for: "Deep Ptych: Subsampled Fourier Ptychography via Generative Priors"

### ISSUE THREAD

**Issue by fahadshamshad** [ml-retrospectives/ml-retrospectives.github.io#15]
## Retrospective for: [*"Deep Ptych: Subsampled Fourier Ptychography via Generative Priors"*](https://arxiv.org/abs/1812.11065)
Paper written by: Fahad Shamshad, Farwa Abbas, and Ali Ahmed. 

Retrospective written by: Fahad Shamshad (fahad.shamshad@itu.edu.pk).

### Paper TL;DR 

We demonstrate the effectiveness of integrating pre-trained generative priors with Fourier Ptychography (FP) problem. FP has recently shown promising results to mitigate the effects of the diffraction blur in long distance imaging. Through extensive numerical simulations, we show that the proposed approach is effective at low subsampling ratios and is highly robust noise perturbations.
  

### Overall Outlook

The paper bridges the gap between deep learning based approaches (that can take advantage of the powerful learned priors) and conventional hand-designed priors such as sparsity (that are flexible enough to handle a variety of model parameters) in the context of FP.

### Opportunities for Improvement

1. **Flaws or mistakes in the paper’s methodology.** 

We did not perform extensive evaluation of the proposed approach, Deep Ptych,  on high-resolution datasets. We just showed one example, that was sampled from the range of the pre-trained generative model (instead of the test set), though we mentioned it in our paper. Effectiveness of the proposed approach on high-resolution dataset requires further investigation. Moreover, we did not train the variational autoencoder (VAE) on the entire MNIST dataset. We picked 5 distinct digits for MNIST and train VAE on that dataset. This was done due to time constraints as the super-resolution stage (non deep learning based method) for super-resolving MNIST digits was time consuming.

2. **Limitations in the applicability of the work.**

There are several limitations of our work that we will address in future.

1. We performed experiments on simulated data instead of real data acquired via actual ptychography setup. The effectiveness of the proposed approach on real data has yet to be explored.

2. Currently, Deep Ptych is limited by the requirement of a large number of fully-observed clean samples of a particular class of images like faces or digits. Unfortunately, obtaining multiple clean samples can be expensive or impractical for many applications including FP.

3. Further, we perform experiments on low-resolution (64x64) images. Recently, generative models have been shown to produce high-resolution and extremely photo-realistic images and we aim to extend our work for high-resolution images using these generative models.

4. We used default parameters of the baseline approaches for comparison and did not optimize the hyperparameters. Optimizing them might lead to improve performance.
